src/test/kotlin/no/nav/familie/tilbake/config/IntegrasjonstestConfig.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.familie.tilbake.config
+
+import no.nav.security.token.support.test.spring.TokenGeneratorConfiguration
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Import
+import org.springframework.context.annotation.Profile
+
+@Profile("integrasjonstest")
```

Comment:
er det samme som "e2e"? Da kan bruke "e2e" som navn istedet. Lett å forstå fordi alle andre applikasjoner bruker det samme

Reply:
Ikke samme som e2e. Dette er for lokale integrasjonstester.